### PR TITLE
Add statistical tools designed for the database

### PR DESCRIPTION
**Pull request by pagreene**
Modify existing tools, and create new tools, to create statistics of the database.

### CODE REVIEW

**Comment by pagreene** [comment]
Here is an example output, as of f93fac5:

```
ext ref statistics:
--------------------
Total number of text refs: 28311743
Total number of refs with content: 18296939
Number of refs with fulltext: 2320956
Number of refs with abstract: 18018599
Number of refs with only abstract: 15975983
Number of refs that have been read: 5159813
Number of refs with fulltext read: 63793

Text Content statistics:
------------------------
Total number of text content entries: 20360715
Latest updates:
    pubmed: 2018-03-16 14:08:55.440271
    manuscripts: 2018-03-13 12:51:47.907932
    pmc_oa: 2018-03-13 12:51:47.228878
Total content read: 5170419
Number of fulltext entries: 2342116
Number of fulltext entries read: 64065
Content by source:
    manuscripts: 483977
    pmc_oa: 1858139
    pubmed: 18018599
Content read by source:
    manuscripts: 23755
    pmc_oa: 78524
    pubmed: 10215504

Reading statistics:
-------------------
Total number or readings: 10317783
Readings by reader version and content source:
    ('61059a-biores-e9ee36',) reading ('pmc_oa',): 7097
    ('61059a-biores-e9ee36',) reading ('pubmed',): 239685
    ('61059a-biores-e9ee36',) reading ('manuscripts',): 1999
    ('1.3.3-61059a-biores-',) reading ('pmc_oa',): 22166
    ('1.3.3-61059a-biores-',) reading ('pubmed',): 4797433
    ('1.3.3-61059a-biores-',) reading ('manuscripts',): 7031
    ('sept14-linux\n',) reading ('pmc_oa',): 27455
    ('sept14-linux\n',) reading ('pubmed',): 381156
    ('sept14-linux\n',) reading ('manuscripts',): 7705
    ('sept14-linux',) reading ('pmc_oa',): 21806
    ('sept14-linux',) reading ('pubmed',): 4797230
    ('sept14-linux',) reading ('manuscripts',): 7020


Statement Statistics:
---------------------
Total number of statments: 5683410
Statements by reader and content source:
        Statements from ('SPARSER',) reading ('pmc_oa',): 54895
    Statements from ('SPARSER',) reading ('pubmed',): 192555
    Statements from ('SPARSER',) reading ('manuscripts',): 28892
    Statements from ('REACH',) reading ('pmc_oa',): 209651
    Statements from ('REACH',) reading ('pubmed',): 1692134
    Statements from ('REACH',) reading ('manuscripts',): 95373

Statements by database:
    biopax: 576847
    entrez: 2788936
    signor: 44042
Number of statements by indra version:
    1.5.0-b'c444d94cf3a3ccfe10c349b96e4e3a3c4633cd2b': 85
    1.5.0-b'c3ecfd187dcc830bc47a2bad42baf94a68e2bfb4': 162810
    1.5.0-b'414595d95ebaa5090b0d656b4ec8e645615784bd': 2
    1.5.0-b'64f4959a4d9dff6fdab2d8e97f9096d0e5e6ec72': 48
    1.5.0-b'41e03125d9370b56f5559ab2afe39d4a506eb33b': 47263
    1.5.0-b'af7e5079873945595cf1e4c1537dc07f637a337d': 20838
    1.5.0-b'1ac9df35cc279afb0a0c551bc63e51b2fc91e2ef': 10181
    1.5.0-b'bfea223a755f52442a575eebd82c47f0c489af6e': 2828
    1.5.0-b'e66d34e642aca6542d3b106bb2fa9f5b14f4059d': 93
    1.5.0-e4fffc040fd63b298fe824a8e2d257431f8e7322: 576847
    1.5.0-b'adaa29aadc5148b0a775463109c185bf91972d37': 66143
    1.5.0-b'1813be253b5028259d947af0d60a6fa400ac2693': 1608277
    1.5.0-b'66fbdfe5d6163e54a55182ddcd76bc3168f14613': 2831
    1.5.0-b'85e3acef99f2f866f55b6000af07944fa8bb0480': 79722
    1.5.0-b'9634d28ba5fd47ee3a8125bc0f4399fefcb14100': 93
    unknown: 2832978
    1.5.0-b'9aa46ab7b9f92d8bc7b6baee33a191125e2c5205': 62
    1.5.0-b'a20efb06cd18379ffbf13a0e02fafe3d977b1acf': 272309
```

File: indra/db/database_manager.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 from __future__ import absolute_import, print_function, unicode_literals
 from builtins import dict, str
+from sqlalchemy.orm.attributes import InstrumentedAttribute
```
Comment (bgyori) [inline indra/db/database_manager.py]: When adding a new import path like this, it often breaks the documentation build (if the parent module is mocked), please add `sqlalchemy.orm` to the mocks list in doc/conf.py and test that the build works.

Comment (pagreene) [reply indra/db/database_manager.py]: `sqlalchemy.orm` is already included in the mock modules. This isn't the first time it's been included. It's used extensively in the content manager.

Comment (bgyori) [reply indra/db/database_manager.py]: It's `sqlalchemy.org.attributes` then, either way it's broken.

Comment (pagreene) [reply indra/db/database_manager.py]: Done

**Comment by bgyori** [comment]
Very nice!

**Pull request merged**